core/pom.xml

```diff
@@ -52,17 +51,6 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
-        <dependency>
-            <groupId>org.liquibase</groupId>
```

Comment:
This should stay. When `dao` will be `<scope>runtime</scope>` you will have to revert this back.